functionnal_tests/cypress/integration/workspace/create_workspace_spec.js

```diff
@@ -42,10 +46,18 @@ describe('Workspace', () => {
           .get('[data-cy="dashboardWorkspaceLabel"]')
           .contains(workspaceTitle1)
 
+        getWorkspaceSidebarChevronIcon(workspaceTitle1)
```

Comment:
I think it's better to check for the submenu in the sidebar.
The chevron is only a graphical item, it doesn't guarantee that the submenu is visible